RouteAnalyzer/build.gradle.kts

```
@@ -2,6 +2,7 @@ plugins {
     kotlin("jvm") version "2.1.10"
     application
     id("com.gradleup.shadow") version "8.3.5"
+    id("org.jetbrains.kotlinx.dataframe") version "0.15.0-RC3"
```

Gradle complains that `ksp` is too old for our version of Kotlin.
This fixes it, but I'm not sure if it is the correct solution.
```suggestion
    id("com.gradleup.shadow") version "8.3.5"
    id("com.google.devtools.ksp") version "2.1.10-1.0.31"
    id("org.jetbrains.kotlinx.dataframe") version "0.15.0-RC3"
```